images/nginx/helpers/x-robots-header-development.conf

```diff
@@ -1,13 +1,9 @@
-# Load environment variable LAGOON_ENVIRONMENT_TYPE into nginx variable. LAGOON_ENVIRONMENT_TYPE will be replaced via envplate during docker-entrypoint
-set $lagoon_environment_type '${LAGOON_ENVIRONMENT_TYPE:-development}';
-
-# By default no x-robots-tag
-set $x_robots_tag '';
-
-# If this is an development environment do not index
-if ($lagoon_environment_type = 'development') {
-  set $x_robots_tag 'noindex, nofollow';
+# Load environment variable LAGOON_ENVIRONMENT_TYPE into nginx variable. LAGOON_ENVIRONMENT_TYPE will be replaced via envplate during docker-entrypo
```

Comment:
as LUA actually has access to the env variables itself, we could now load the env variables directly instead via the envplate system in the docker-entrypoint which is hard to understand.

be careful though with: https://stackoverflow.com/questions/41800071/unable-to-use-environment-variables-in-lua-code